packages/nx-plugin/src/schematics/e2e-project/files/tests/__pluginName__.test.ts__tmpl__

```diff
@@ -16,29 +16,4 @@ describe('<%= pluginName %> e2e', () => {
 
         done();
     })
-
-    describe('--directory', () => {
-      it('should create src in the specified directory', async (done) => {
-        const plugin = uniq('<%= pluginName %>');
-        ensureNxProject('<%= npmPackageName %>', '<%= pluginOutputPath %>');
-        await runNxCommandAsync(
-          `generate <%=npmPackageName%>:<%= pluginPropertyName %> ${plugin} --directory subdir`
-        );
-        expect(() => checkFilesExist(`libs/subdir/${plugin}/src/index.ts`)).not.toThrow();
-        done();
-      });
-    });
-
-    describe('--tags', () => {
-      it('should add tags to nx.json', async (done) => {
-         const plugin = uniq('<%= pluginName %>');
-        ensureNxProject('<%= npmPackageName %>', '<%= pluginOutputPath %>');
-        await runNxCommandAsync(
-          `generate <%=npmPackageName%>:<%= pluginPropertyName %> ${plugin} --tags e2etag,e2ePackage`
-        );
-        const nxJson = readJson('nx.json');
-        expect(nxJson.projects[plugin].tags).toEqual(['e2etag', 'e2ePackage']);
-        done();
-      });
-    });
```

Comment:
Any particular reason why these were removed?

Reply:
I'm actually not sure, I must have been debugging something. I can definitely add these back.